dev-client/src/screens/HomeScreen/components/LandPKSInfo.tsx

```diff
@@ -56,11 +51,11 @@ export const LandPKSInfo = () => {
         <Column>
           {[1, 2, 3].map(index => (
             <HStack key={index}>
-              <Text variant="body1" mr={2}>
-                {index + 1}
+              <Text variant="body1" mr={numberedListMargin}>
```

Comment:
suggestion: can we use https://github.com/jsamr/react-native-li? This looks useful for bulleted lists, too.